src/main/java/com/maxmind/geoip2/model/AbstractCityResponse.java

```diff
@@ -14,6 +14,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * This class represents the GeoIP2 City response model.
```

Comment:
Maybe something like:

```suggestion
 * Abstract class for models that contain City data.
```